### DISCUSSION TITLE
DEAR CARSON I CANNOT DOWNLOAD OR INSTALL ANY MODEL WITH VERSION 009

### DISCUSSION THREAD

**Discussion by OculusQuestVR** [carson-katri/dream-textures#461] [General]
Dear Carson, I'm Mauro from Rome. Congratulations for the excellent work you did with Dream Textures for Blender, I would love to be able to use it and test its potential but unfortunately, despite having correctly installed version 009, it is impossible for me to download any model either directly or from Hugging Face on which I am registered (I always the following error "DLL load failed while importing _psutil_windows: The specified module could not be found".
I tried to install the psutil_windows module with pip in the folder but it still gives me the same error.
Can I fix it somehow? Can you help me?
Can I eventually import the model in ckp format directly into the Dream Texture folder? If yes, where should I copy it? Thanks in advance for your time
Greeting from Rome
Mauro
![For Carson](https://user-images.githubusercontent.com/112269600/208628634-f0b1e48d-8e6b-4135-b940-0bd82f203dd1.png)

**Comment by Alzwei** [comment]
Windows 11, Blender 3.4 , same problem. When installing dependencies, console write: "PermissionError: [WinError 5] Access is denied: 'C:\\Users\\alex_\\AppData\\Roaming\\Blender Foundation\\Blender\\3.4\\scripts\\addons\ \ dream_textures\\.python_dependencies\\google\\protobuf\\internal\\_api_implementation.cp310-win_amd64.pyd'". But packages can be different, several times "psutil"

**Comment by mbrusevics** [comment]
Seams like I have the same kind of problem with Blender 3.3 
![image](https://user-images.githubusercontent.com/63247244/208659744-3ae08f85-c31d-4069-8fc3-44e97f61ad1c.png)

**Reply by mbrusevics** [reply]
But in my case no errors at all. It just nothing heppends

**Reply by mbrusevics** [reply]
Cured by proper reinstanig Blender.

**Comment by OculusQuestVR** [comment]
Dear all, I solved it by reinstalling Blender from scratch (version 3.4). The Stable Diffusion model to use is the one with the depth map.
However, the fbx created are not exported with the textures (even putting the Copy option with the textures embedded), can anyone tell me how to fix it?

Thank you
Mauro

**Comment by OculusQuestVR** [comment]
Has anyone successfully exported an fbx model with the textures created with Dream Texture displayed correctly?
I've tried everything but can't see them in any way. The created textures normally export without problems. Has anyone solved it?

Mauro
![screen2](https://user-images.githubusercontent.com/112269600/208869041-a1e65405-adea-41aa-8e7b-6b12bf89c430.png)
![screen1](https://user-images.githubusercontent.com/112269600/208869046-b98bb0f7-8f74-4ac7-9382-ff546093e1b6.png)

**Comment by OculusQuestVR** [comment]
There is no way to export the materials and textures created with Dream Textures to Unity as a fbx model, the materials are always white (empty).
Has anyone solved it by any chance?

Mauro

**Reply by JasonHoku** [reply]
> There is no way to export the materials and textures created with Dream Textures to Unity as a fbx model, the materials are always white (empty). Has anyone solved it by any chance?

Make sure the textures are first saved to a non temp path on disk, then when exporting as an fbx make sure to select "Copy" for the Path Mode and activate it. This should pack the fbx with the right info for unity, there might be options you need to select in unity when importing too otherwise once you've saved the textures to disc you can also just drag them into unity and onto your objects as long as the uv's are mapped properly:
![image](https://user-images.githubusercontent.com/6715569/209138110-31e744ee-f82b-464c-bef5-53573ccc7c9d.png)
